packages/frontend/src/ee/features/aiCopilot/components/SuggestedQuestions/SuggestedQuestions.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+    Paper,
+    SimpleGrid,
+    Stack,
+    Text,
+    UnstyledButton,
+} from '@mantine-8/core';
+import shuffle from 'lodash/shuffle';
+import { type FC, useMemo } from 'react';
+import classes from './SuggestedQuestions.module.css';
+import { getLeastSimilar } from './getLeastSimilar';
+
+type SuggestedQuestion = {
+    question: string;
+    uuid: string;
+};
+
+type SuggestedQuestionsProps = {
+    questions: SuggestedQuestion[];
+    onQuestionClick: (question: string) => void;
+    isLoading?: boolean;
+};
+
+const MIN_AVAILABLE_QUESTIONS = 6;
+const MIN_SUGGESTED_QUESTIONS = 2;
```

Comment:
I'm thinking this is a bit too high - wdyt of 
min available -> 3
min suggested -> 1

Reply:
I like having at least 2, we can adjust

Reply:
cool, just wondering how long it will take to see this feature considering it's all very new